### ISSUE TITLE
Accessibility: buttons marking icon missing aria label

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt_framework#2293]
The `<div class="buttons-marking-icon">` element in buttons.hbs is missing the aria label `aria-label="{{#if _isCorrect}}{{@root/_globals._accessibility._ariaLabels.answeredCorrectly}}{{else}}{{@root/_globals._accessibility._ariaLabels.answeredIncorrectly}}{{/if}}"` that is found on another, same named, element further down the file.

Action: Copy aria-label onto both `<div class="buttons-marking-icon">` elements
